mollie-payments-for-woocommerce/includes/mollie/wc/autoload.php

```diff
@@ -59,9 +59,9 @@ public static function autoload ($class_name)
             }
         }
         // Mollie API client
-        elseif (stripos($class_name, "Mollie_API_") === 0)
+        elseif (stripos($class_name, "Mollie") === 0)
         {
-            $class_path = $base_path . '/mollie-api-php/src/' . str_replace('_', '/', $class_name) . '.php';
+            $class_path = $base_path . str_replace( 'Mollie/API', '/mollie-api-php/src', str_replace('\\', '/', $class_name)) . '.php';
```

Comment:
Today I pulled the latest changes and copied them to my WordPress installation. Now I got the message "Fatal error: Uncaught Error: Class 'Mollie\Api\MollieApiClient' not found". 

After searching I found out you are using `Mollie\API` in the usages of the Mollie API client but this is incorrect and will break the autoloader because the client uses [`Mollie\Api`](https://github.com/mollie/mollie-api-php/blob/master/src/MollieApiClient.php#L3). It uses the [PSR-4 autoloader standard](https://www.php-fig.org/psr/psr-4/).

PHP class names are case insensitive so this shouldn't be a problem but because the class name is used in the autoloader to translate to a file name this will be a problem on platform where the file system is case sensitive.

I'm not sure why I didn't have this problem before the last git pull but let's make sure this works everywhere:

- Make sure to use the correct classnames: Replace all `Mollie\API` with `Mollie\Api`.
- Replace `Mollie/API` with `Mollie/Api` on this line in the autoloader.

Reply:
This doesn't completely solve the problem because no autoloader is defined for the Mollie API client (Composer) dependencies (e.g. The Guzzle client). Previously we had registered the Mollie API client autoloader in `mollie-payments-for-woocommerce.php`. In the latest commit [it was removed](https://github.com/mollie/WooCommerce/pull/214/commits/5b2eaedfd22ab207236c3f722a9b40446d16c1cd), this is also the reason why everything breaks for me now.

You need to add it back by adding this to `mollie-payments-for-woocommerce.php` on line 25:
`require_once 'includes/mollie-api-php/vendor/autoload.php';`

This will register an autoloader for the Mollie API client (`Mollie\Api\*`) and it's dependencies. You can [remove this](https://github.com/mollie/WooCommerce/blob/5b2eaedfd22ab207236c3f722a9b40446d16c1cd/mollie-payments-for-woocommerce/includes/mollie/wc/autoload.php#L61-L70) from the plugin's autoloader.

Reply:
@lvgunst 
Oh shoot, already found that I made a mistake when merging master yesterday, didn't commit it, this line was missing in `mollie-payments-for-woocommerce.php` on line 25.
``` require_once 'includes/mollie-api-php/vendor/autoload.php'; ```

Reply:
We use git submodules to include the Mollie API client. After running `git submodule update` you have to run `composer install` to include the Composer dependencies. The `mollie-api-php/vendor` will be part of the plugin bundle. I think you can decrease the size by only installing the non-dev dependecies (`composer install --no-dev`) but you have to test this.

Reply:
Just committed the change.

Reply:
> You can remove this from the plugin's autoloader.
Also done!

Reply:
You only need to fix the usages of the class names. Replace `Mollie\API` with `Mollie\Api`.

Reply:
Yes, global search & replace, just pushed it.

Reply:
Really appreciate your review by the way!